Gandalan.IDAS.WebApi.Client/Settings/ConnectHub.cs

```diff
@@ -19,12 +25,13 @@ public async Task<HubResponse> GetEndpoints(string apiVersion = null, string env
                 (clientOs != null ? "clientOS=" + clientOs : "");
             try
             {
-                var response = await new HttpClient().GetStringAsync(requestUrl);
-                return JsonConvert.DeserializeObject<HubResponse>(response);
+                using var httpClient = new HttpClient();
+                var response = httpClient.GetStringAsync(requestUrl).GetAwaiter().GetResult();
```

Comment:
use await?

Reply:
await here will "fire and forget" this thread if we will call `WebApiConfigurations.Initialize()` from constructors. This is the main reason I created this PR - to fix IBOS2 Legacy.

You can read more here: https://stackoverflow.com/questions/57999839/httpclient-getstringasync-is-not-executed

Reply:
I see - why not make the Main method async as mentioned in one of the replies?